packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div :class="['c-nav-container', { 'is-visible': isOpen }]">
+        <div :class="$style['c-countrySelector']">
+            <header :class="$style['c-countrySelector-header']">
+                <f-button
+                    button-type="ghost"
+                    is-icon
+                    :class="$style['c-countrySelector-header-button']"
+                    :aria-label="copy.countrySelector.goBackToMainMenu"
+                    @click="$emit('goBackButtonClick')">
+                    <arrow-icon :class="$style['c-countrySelector-goBackIcon']" />
+                </f-button>
+
+                <h3>{{ copy.countrySelector.selectYourCountryText }}</h3>
+            </header>
+
+            <ul
+                class="c-nav-popoverList c-nav-popoverList--twoColumns"
+                data-test-id="c-countrySelector-list">
+                <li
+                    v-for="(country, i) in countries"
+                    :key="i + '_Country'"
```

Comment:
Might be good to use `country.key` in the key here rather than `index` as it allows Vue to optimise DOM updates if these items ever change. We were but by this on the menu recently where text would update in items but images remained in the wrong place. 

It's probably not much of an issue here as the data is static but it's a good practice to follow.

```suggestion
                    v-for="country in countries"
                    :key="country.key + '_Country'"
```

Reply:
The problem with having `country.key` here is that we have `ca` and `ch` key used multiple times as we show separate links for different languages for Canada and Switzerland. We use `country.key` for showing correct flags

Reply:
I can specify separate keys for flags and have unique keys for every link

Reply:
Right, yeah I see. I'll leave it up to you if you want to make this change or not 😺

Reply:
Actually, I have a next PR ready to come, so I will include this change to that one

Reply:
I guess we could potentially use the culture as a key? i.e., `en-CA` and `fr-CA`.